### ISSUE TITLE
Petridish provides wrong branch information

### ISSUE THREAD

**Issue by kptdobe** [adobe/petridish#64]
In a fresh project, just add the `action` parameter to the `pre` function and output `action.request.params.ref`.

Code could look like this:
```
function pre(payload, action) {
  payload.content.time = `${new Date()}`;
  console.log(action.request.params.ref);
}
module.exports.pre = pre;
```

Run `hlx up`
Verify that the output is `master`.

Checkout a new branch: `git checkout -b test`
Run `hlx up`

-> the output is still `master` while it should be `test`.

While this is not an issue for default / simple use of the dev environment, this is an issue when starting to use the owner / repo / ref params to request extra stuff like in the nav case of helix-helpx: the code computes an git API call with those params and expect to be in the same "branch". 

Nothing fails, just different content from another branch!

I lost quite some time today before realising my code was correct but the api result was the one from the master branch and not my current working branch!

**Comment by kptdobe** [comment]
I traced it to https://github.com/adobe/petridish/blob/master/src/HelixProject.js#L247: we initialise the giturl without providing a branch, i.e. always defaults to `master`. There should be somewhere the capability to retrieve the current branch: I assume this should be provided by the gitserver. @stefan-guggisberg WDYT ?

**Comment by stefan-guggisberg** [comment]
@kptdobe agreed, we'll need to be able to determine the currently checked-out branch. i'll look into it.

**Comment by stefan-guggisberg** [comment]
fixed in `v1.4.6` (aa7f72fdd5aac7902bd2e1a8a031087930cad63e)

i've also published `helix-cli` `v0.7.11` which depends on `petridish` `v1.4.6`
